include/rocksdb/listener.h

```diff
@@ -261,6 +261,8 @@ struct ExternalFileIngestionInfo {
   SequenceNumber global_seqno;
   // Table properties of the table being flushed
   TableProperties table_properties;
+  // Level that the external file is ingested into
+  int picked_level;
```

Comment:
is this redundant?

Reply:
No, need this info to collect the ingestion level. It's used in the `on_ingest_external_file` hook of tikv.